Source/Core/Common/StringUtil.h

```diff
@@ -47,20 +47,60 @@ std::string ArrayToString(const u8* data, u32 size, int line_len = 20, bool spac
 std::string_view StripSpaces(std::string_view s);
 std::string_view StripQuotes(std::string_view s);
 
+std::string ReplaceAll(std::string result, std::string_view src, std::string_view dest);
+
 bool TryParse(const std::string& str, bool* output);
-bool TryParse(const std::string& str, u16* output);
-bool TryParse(const std::string& str, u32* output);
-bool TryParse(const std::string& str, u64* output);
 
-template <typename N>
-static bool TryParse(const std::string& str, N* const output)
+template <typename T, std::enable_if_t<std::is_integral_v<T> || std::is_enum_v<T>>* = nullptr>
+bool TryParse(const std::string& str, T* output)
 {
-  std::istringstream iss(str);
-  // is this right? not doing this breaks reading floats on locales that use different decimal
-  // separators
-  iss.imbue(std::locale("C"));
+  char* end_ptr = nullptr;
+
+  // Set errno to a clean slate.
+  errno = 0;
+
+  // Read a u64 for unsigned types and s64 otherwise.
+  using ReadType = std::conditional_t<std::is_unsigned_v<T>, u64, s64>;
+  ReadType value;
 
-  N tmp;
+  if constexpr (std::is_unsigned_v<T>)
+    value = strtoull(str.c_str(), &end_ptr, 0);
```

Comment:
```suggestion
    value = std::strtoull(str.c_str(), &end_ptr, 0);
```

`<cstdlib>` needs to be included, too